src/theme/variables.scss

```diff
@@ -33,9 +33,13 @@ $app-direction: ltr;
 // colors so you can add, rename and remove colors as needed.
 // The "primary" color is the only required color in the map.
 
+.item-content {
+  background-color:#ffffff;
+}
+
 $colors: (
-  primary:    #488aff,
-  secondary:  #32db64,
+  primary:    #e70f0f,
```

Comment:
code colour of "primary" is not "#e70f0f" please change code to "#189DAE"

Reply:
before Create New Pull Request , please change "base: master" to "base:develope"